docs/defects-api.yml

```
@@ -39,6 +38,69 @@ paths:
           description: Access token does not have the required scope
         "404":
           description: Defects not found
+
+  /defects/iva/getDefectsForVehicle:
```

Are we okay with this naming with get in the URL and not the standard?
 ```suggestion
   /defects/iva:
```

---

Would /defects/iva/vehicle be more preferred?

---

Like this change a lot :)

---

As in what you have done, not the /vehicle, sorry!